docs/changehistory/NextVersion.md

```diff
@@ -80,6 +81,10 @@ Below is a list of the changes from this move, some of these new access point ma
  UiFramework.childWindowManager.closeChildWindow | UiFramework.childWindows.close
  UiFramework.backstageManager | UiFramework.backstage
 
+### ViewSelector Enhancements
+
+Search functionality has been added to the `ViewSelector` component, allowing for additional ease when navigating between iModel views. Search functionality is optional and enabled by default. To disable search functionality, set the `searchBox` property of `ViewSelector` to false.
```

Comment:
```suggestion
Search functionality has been added to the [ViewSelector]($appui-react) component, allowing for additional ease when navigating between iModel views. Search functionality is optional and enabled by default. To disable search functionality, set the [ViewSelector.searchBox]($appui-react) to `false`.
```

Reply:
Consider using doc links so people can quickly jump to more information on the doc site.

Reply:
I guess by "searchBox property of ViewSelector" you must have meant ViewSelectorProps, right?

Reply:
Yes, that is correct